test/middleware/response_validation_open_api_3_test.rb

```diff
@@ -17,6 +17,22 @@ def app
     assert_equal 200, last_response.status
   end
 
+  it "passes through a valid response with content-type (lower-case)" do
+    status = 200
+    headers = { "content-type" => "application/json" }
+    response = JSON.generate(CHARACTERS_RESPONSE)
+
+    @app = Rack::Builder.new {
+      use Committee::Middleware::ResponseValidation, {schema: open_api_3_schema}
+      run lambda { |_|
+        [status, headers, [response]]
+      }
+    }
```

Comment:
I avoid touching the method `new_response_rack` to keep this PR small.
https://github.com/interagent/committee/blob/8e3e43d3494bff5a1a15d28f4b33e5cc0208d7b3/test/middleware/response_validation_open_api_3_test.rb#L285-L295

The method will be refactored after this PR is merged.